Add changelog entry for Linear OAuth token refresh fix (#520)
Promptless/docs: src/content/changelog/changelogs/may-2026.mdx

**Bug Fixes:**

* **Bot-Authored Commit Evaluation:** Fixed Promptless skipping GitHub commits authored by bot accounts—such as paper-style editors and headless writers that push human-written content under a bot identity—regardless of the diff. Promptless now evaluates these commits based on the diff content, just as it does for human-authored commits. PR and issue comments from bot accounts remain filtered as before.

> * **Linear Integration Token Refresh:** Fixed Linear OAuth tokens expiring and causing silent integration failures. Tokens now refresh automatically before expiration. If you connected Linear before this fix, your integration will show "Reconnect required" in the dashboard—disconnect and reconnect Linear once to restore full functionality. New connections work seamlessly.

Reviewer: ```suggestion
* **Linear Integration Token Refresh:** Fixed Linear OAuth tokens expiring and causing silent integration failures. Tokens now refresh automatically before expiration. If you connected Linear before this fix, you need to reconnect Linear to restore full functionality. New connections work without issue.
```
